### DISCUSSION TITLE
Ignore dock - Fill all available screen height/width

### DISCUSSION THREAD

**Discussion by unlocomqx** [rxhanson/RectanglePro-Community#70] [Ideas]
Hi,

I'd like my apps to take all the screen height
Currently, the screen height is reduced by the dock height

<img width="1792" alt="screenshot-000565" src="https://user-images.githubusercontent.com/793712/171624848-11a33980-d512-4f8d-b89c-45aa4ddead74.png">
As you can see, it would be convenient for me to get more height for my IDE and browser

Is it possible to add an option to make the apps take all height (or all width if the dock is to the left/right)

Thank you

**Comment by rxhanson** [comment]
This is a macOS limitation. I can move windows behind the dock, but not if they are 100% height. MacOS will just position it above the dock if I try to make it 100% height.

**Comment by unlocomqx** [comment]
It's cool, thank you

**Comment by llumiaho** [comment]
I'm running macOS 12.6.1 and it seems that now windows maximize behind the dock which I don't want. I wonder if it is a bug or feature. I was hoping that Rectangle could help with this but I don't see an option for this.

**Reply by llumiaho** [reply]
ugh, functionality was restored by restarting the computer

**Reply by unlocomqx** [reply]
The classic on and off fix
